week-one/Course.java

```diff
@@ -67,23 +68,23 @@ public boolean isNanoInTheClass(Nano student) {
 
   public ArrayList<Nano> rosterSortedAlphabetically() {
```

Comment:
Can you optimize this method so the worst-case scenario big-o is at least `n^2`?  Currently, the worst-case scenario is `n^2 + n`. However, on average it will be `n^2` since the second loop won't matter.